common/src/main/scala/explore/cache/CacheComponent.scala

```diff
@@ -15,9 +16,9 @@ import japgolly.scalajs.react.vdom.html_<^.*
 trait CacheComponent[S, P <: CacheComponent.Props[S]: Reusability]:
   private type F[T] = DefaultA[T]
 
-  protected val initial: P => F[S]
+  protected val initial: (P, (S => S) => F[Unit]) => F[S]
```

Comment:
Can I request a comment explaining the type?

Reply:
Sure. I also realized I forgot to rename the parameter in ProgramCache. At the time I was fresh out of good names and just wanted to get it working. I'll change that, too.